completion/available/notify-send.completion.bash

```diff
@@ -0,0 +1,41 @@
+# shellcheck shell=bash
+<<<<<<< HEAD
```

Comment:
Looks like this file has some merge conflicts...

Reply:
Done